src/scripts/crowdin/import/utils.ts

```diff
@@ -0,0 +1,171 @@
+import { copyFileSync, existsSync, mkdirSync, readdirSync } from "fs"
+import { join } from "path"
+
+import i18Config from "../../../../i18n.config.json"
+import { DOT_CROWDIN } from "../translations/constants"
+
+import { BucketsList, SelectionItem, SummaryItem, TrackerObject } from "./types"
+
+/**
+ * Some language codes used in the repo differ from those used by Crowdin.
+ * This is used to convert any codes that may differ when performing folder lookup.
+ */
+export const getCrowdinCode = (code: string): string =>
```

Comment:
Shouldn't we move some of these (if not all) functions to the `@/lib/utils/crowdin` file?

I'd probably only keep the summary logic here since it is tightly related to this script.

Reply:
@pettinarip If we do this we just need to make sure we don't import dependencies using `@` syntax anywhere in the chain... In the `src/scripts` folder we are typically running these with `ts-node` which fails when it encounters that shorthand with `code: 'MODULE_NOT_FOUND'` (at least with our current setup). 

In `src/lib` we use the `@` import syntax, so if we want to mix them we may need to reverse that syntax in some cases.

Reply:
Got it! didn't know about that problem.